proteus/mprans/Kappa.py

```diff
@@ -136,7 +136,7 @@ def __init__(self,LS_model=None,V_model=0,RD_model=None,dissipation_model=None,M
         self.variableNames=['kappa']
         nc=1
         self.nd = nd
-        assert self.nd == 3, "Kappa only implements 3d for now" #assume 3d for now
+        #assert self.nd == 3, "Kappa only implements 3d for now" #assume 3d for now
```

Comment:
It did not work very well during 2D simulations, with the time step becoming increasingly small, until the simulation crashes. I will uncomment the line before merging.